sympy/functions/elementary/tests/test_hyperbolic.py

```diff
@@ -593,6 +609,9 @@ def test_atanh():
     assert atanh(-1) == -oo
 
     # at infinites
+    assert atanh(oo) == -I*pi/2
+    assert atanh(-oo) == I*pi/2
```

Comment:
There is a `def test_atanh_infinities():`  in the same file. You should remove that if you are putting these here.